release-notes/Web_apps/Web_apps_Feature_Release_10.5/Web_apps_Feature_Release_10.5.2.md

```diff
@@ -30,21 +30,9 @@ Up to now, all GQI-related operations were executed by the SLHelper process, whi
 - System load by GQI operations can now be balanced among the different DataMiner Agents in a DMS.
 
 > [!IMPORTANT]
-> Until further notice, the DataMiner web apps will continue to use the SLHelper process to execute GQI-related operations. If you want them to use the new GQI DxM instead, update the *C:\\Skyline DataMiner\\Webpages\\web.config* file accordingly.
+> Until further notice, the DataMiner web apps will continue to use the SLHelper process to execute GQI-related operations. If you want them to use the new GQI DxM instead, update the *C:\\Skyline DataMiner\\Webpages\\API\\Web.config* file accordingly.
 
-##### Specifying when idle child processes should be terminated
```

Comment:
If at all possible, I would prefer to keep this information in the release notes as well. This because when a user looks at the release notes for an older DataMiner version, those are supposed to show the info as it was at the time of publication; but if we replace the info with a link to a section that will change over time, it will become very difficult for the user to still know exactly what the feature is like in that particular version.

Reply:
Ok I'll put it back